scripts/drtprod

```diff
@@ -86,6 +86,19 @@ case $1 in
         # setup dns
         $0 dns drt-main create
         ;;
+      "drt-chaos")
+        roachprod create drt-chaos \
+          --clouds gce \
+          --gce-managed \
+          --gce-zones "us-east1-b" \
+          --nodes 5 \
+          --gce-machine-type n2-standard-8 \
+          --gce-pd-volume-size 3000 --local-ssd=false \
```

Comment:
PD and not local ssd?

Reply:
We could do local ssd to save $ etc, but I feel like pd has been more reliable for us. Good with either though.

Reply:
Let's stick with pd due to the ongoing issues.